### ISSUE TITLE
[REFACTOR] Consensus chromatogram processing

### ISSUE THREAD

**Issue by singjc** [Roestlab/massdash#45]
Current implementation for averaging, median and percentile_averaging consensus chromatograms can be found here: [chromatogram_data_handling.py#L246](https://github.com/Roestlab/massseer/blob/dev/massseer/chromatogram_data_handling.py#L246)

**Comment by jcharkow** [comment]
This is completed in previous major refactoring

**Comment by singjc** [comment]
> This is completed in previous major refactoring

Yip, the backend code is complete, but not yet implemented in the GUI. If it's still something we want part of the GUI that is.

**Comment by jcharkow** [comment]
Yes I think we still want it in the GUI however it is not a major priority
